sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -14346,60 +14487,38 @@
                 "step": "any",
                 "integer": ""
               },
-              "key": "totalUnsubsidizedDayCareCostsDuringClassHoursForTheProposedStudyPeriodForDisabledInfirmedDependents12YearsOfAgeAndOlder",
+              "unique": false,
+              "errorLabel": "",
+              "key": "daycareCosts12YearsOrOver",

```

Comment:
This was a change I did so it is part of your form definition upload. I will not upload a separate definition